src/Common/Admin/Entities/Link.php

```diff
@@ -27,25 +29,32 @@ class Link extends Base_Entity {
 	private string $url = '';
 
 	/**
-	 * The text for the link.
+	 * Content for the link if not a string.
 	 *
-	 * @var string
+	 * @var ?Base_Entity
 	 */
-	private string $text = '';
+	private ?Base_Entity $content = null;
```

Comment:
I'm not sure what's going on here, but this is confusing. Is it a string?
Can it be a string?